src/components/LobLabel/LobLabel.stories.js

```diff
@@ -0,0 +1,55 @@
+import LobLabel from './LobLabel.vue';
+import TextInput from '../TextInput/TextInput.vue';
+import mdx from './LobLabel.mdx';
+
+export default {
+  title: 'Components/LobLabel',
+  component: LobLabel,
+  parameters: {
+    docs: {
+      page: mdx
+    }
+  },
+  argTypes: {
+    label: {
+      control: {
+        type: 'text'
+      }
+    }
+  }
+};
+
+const WithTextInputTemplate = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { LobLabel, TextInput },
+  setup: () => ({ args }),
+  template: `
+    <lob-label v-bind="args"/>
+    <text-input id="1" placeholder="Name" label="Name" srOnlyLabel/>
```

Comment:
For the sake of being able to see the component and its styling in Storybook, can we remove `srOnlyLabel` from both of these stories?

Reply:
This is where I get a little confused about what the story is supposed to show - the TextInput has a label by default, so without the `srOnlyLabel` we see 2 labels. Should I remove `<lob-label/>`?

Reply:
You should use a vanilla input. Storybook serves as documentation for folks within Lob to see what our components look like and how to use them. In the case of a LobLabel, we want to show someone how to use it if they wanted to create a component within their own project that has a label consistent with our design system. We already show how to use a TextInput in the TextInput story so we don't want to use that here. 
```
<input type=text />
```